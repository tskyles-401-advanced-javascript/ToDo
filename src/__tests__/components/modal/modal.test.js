import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import React from 'react';
import renderer from "react-test-renderer";
import Modal from '../../../components/modal/index';
Enzyme.configure({ adapter: new Adapter() });

describe('<Modal />', () => {
  it('is alive on start', () => {
    let app = shallow(<Modal/>);

    expect(app.find('header').exists()).toBeTruthy();
  });

  it('renders correctly', () => {
    const tree = renderer.create(<Modal/>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});