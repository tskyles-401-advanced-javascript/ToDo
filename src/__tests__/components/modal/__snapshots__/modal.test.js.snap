// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Modal /> renders correctly 1`] = `
<div
  className="modal"
>
  <div>
    <header>
      <span
        className="title"
      />
      <button>
        X
      </button>
    </header>
    <div />
  </div>
</div>
`;
